CHANGELOG.md

```diff
@@ -3,6 +3,89 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.2.0
+
+_Apr 12, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Support reordering of items in `SimpleTreeView` (#12369) @flaviendelangle
```

Comment:
Just realized that the PR title could be misunderstood, people could think that we support drag&drop on this component.
I would remove the highlight (even if it was a big task) and rename the PR "Fix behaviors when the item order changes"